vnext/src/IntegrationTests/AsyncStorageTest.windesktop.js

```diff
@@ -71,7 +89,7 @@ function testSetAndGet() {
     AsyncStorage.getItem(KEY_1, (err2, result) => {
       expectAsyncNoError('testSetAndGet/getItem', err2);
       expectEqual(result, VAL_1, 'testSetAndGet setItem');
-      updateMessage('get(key_1) correctly returned ' + result);
+      updateMessage('get(key_1) correctly returned ' + String(result));
```

Comment:
>String [](start = 55, length = 6)

I am just curious why is it needed: concatenation with a string literal will force the conversion to string anyway.

Reply:
Would it be better to use the `stringify` function here and in the next case below.

---
In reply to: [558392447](https://github.com/microsoft/react-native-windows/pull/6883#discussion_r558392447) [](ancestors = 558392447)

Reply:
Note, this file is a slightly patched version of the original provided by React Native.

It is the intention to keep it as close to equal as possible, regardless of the clear areas of improvement.